src/Adapter/Environment.php

```diff
@@ -85,4 +85,12 @@ public function isDebug()
     {
         return $this->isDebug;
     }
+
+    /**
+     * @inheritDoc
+     */
+    public function getCacheDir()
+    {
+        return _PS_ROOT_DIR_.'/var/cache/'.$this->getName(). DIRECTORY_SEPARATOR;
```

Comment:
Mixin `/` and `DIRECTORY_SEPARATOR` const is useless because `/` works on windows, unix and mac.